src/plugins/router/routes/generic/health.js

```diff
@@ -0,0 +1,19 @@
+const { ActionTransport } = require('../../../../constants');
+
+function GenericHealthCheck(): Object {
+  return {
```

Comment:
I'd make the action a bit more interesting:

1. allow each plugin to report health status (ie redis, amqp, etc) or define health-status getters in the plugins
2. whenever the route is "triggered" - ask all status getters and if they all report success - we are good
3. if not - define whether status is critical or not (ie how many times it should report as failed)
  - report this via service.log (goes to sentry when enabled)
4. when we get "critical" threshold - reply with status code 500 or something